projects/plugins/jetpack/_inc/client/components/jetpack-product-card/style.scss

```diff
@@ -77,67 +77,44 @@
 }
 
 .jp-product-card__price {
-	display: flex;
-	flex-wrap: wrap;
-	flex-direction: row;
-	margin-top: auto;
-	color: $black;
-}
-
-.jp-product-card__raw-price {
-	display: flex;
-	margin: 0 22px 0 0;
-	font-size: $font-headline-medium;
-}
+	// Price item
+	> div > p {
+		margin-inline-end: 0.75rem !important;
+
+		font-size: 3.375rem;
```

Comment:
These values were taken from the prices in the Assistant section, for consistency.